src/fsharp/CompileOptions.fsi

```diff
@@ -98,3 +98,4 @@ val DoWithErrorColor : bool -> (unit -> 'a) -> 'a
 val ReportTime : TcConfig -> string -> unit
 val GetAbbrevFlagSet : TcConfigBuilder -> bool -> Set<string>
 val PostProcessCompilerArgs : string Set -> string [] -> string list
+val ParseCompilerOptions : (string -> unit) * CompilerOptionBlock list * string list -> unit
```

Comment:
This function is used in implementing some improvements to the F# compiler service API so making it available in the signature here